kubernetes/resource_kubernetes_ingress_test.go

```diff
@@ -164,7 +163,7 @@ func TestAccKubernetesIngress_WaitForLoadBalancerGoogleCloud(t *testing.T) {
 				Config: testAccKubernetesIngressConfig_waitForLoadBalancer(name),
 				Check: resource.ComposeAggregateTestCheckFunc(
 					testAccCheckKubernetesIngressExists("kubernetes_ingress.test", &conf),
-					resource.TestCheckResourceAttrSet("kubernetes_ingress.test", "load_balancer_ingress.0.ip"),
+					resource.TestCheckResourceAttrSet("kubernetes_ingress.test", "status.0.l.0.ip"),
```

Comment:
TC is saying this might have introduced a flake:

```
=== RUN   TestAccKubernetesIngress_WaitForLoadBalancerGoogleCloud
    TestAccKubernetesIngress_WaitForLoadBalancerGoogleCloud: resource_kubernetes_ingress_test.go:156: Step 1/1 error: Check failed: 1 error occurred:
        	* Check 2/2 error: kubernetes_ingress.test: Attribute 'status.0.l.0.ip' expected to be set
```

But since I like working with the tests so much, I can probably poke at it and get it fixed in another PR.

Reply:
Whoops, this change wasn't actually supposed to be in this PR 😄